### DISCUSSION TITLE
What is the correct build.gradle.kts

### DISCUSSION THREAD

**Discussion by watermelone1** [Ayfri/Kore#47] [Q&A] [answered]
The build.gradle.kts file I have is [here](https://appdevtools.com/pastebin/cnPggk)

(I do not use kotlin very often and mostly use java)

**Comment by Ayfri** [comment, marked as answer]
Hello, I think the issue with your configuration is that you are using the wrong version of Kotlin, try using the latest version `1.9.20`.
